### ISSUE TITLE
Fathom tracking partially unblocked

### ISSUE THREAD

**Issue by JobcenterTycoon** [AdguardTeam/cname-trackers#41]
Hello, while most of the Fathom tracking is already blocked in https://github.com/AdguardTeam/cname-trackers/blob/master/trackers/fathom.txt a few scripts pass through because they use a other CNAME domain.

Examples:

`prefinery.com` -> `electric-eighten.b-cdn.net`
`steadybit.com` -> `type-thirtyseven.b-cdn.net`
`meinsmartplan.de` -> `gorgeous-easy.b-cdn.net`

Example for the script: `prefinery.com` -> `https://glowing-van.prefinery.com/script.js` But the list only contains `ant.prefinery.com`

I don’t know if it can be fixed because `b-cdn.net` get widely used

**Comment by ameshkov** [comment]
Yeah, `b-cdn.net` shouldn't be blocked. They might be using it as a proxy.

**Comment by GeQi** [comment]
Yes, [Fathom explicitly states that they use Bunny as proxy](https://usefathom.com/features/eu-isolation). This looks rather hopeless, is there anything that can be done about this?

**Comment by GeQi** [comment]
Reported to the filter list repo: https://github.com/AdguardTeam/AdguardFilters/issues/116645

**Comment by ameshkov** [comment]
Actually, I think we can solve this rather easily by adding one more step to the detection algorithm.

We'll need to scan all bunny CDN cnames and check if the domain hosts Fathom's javascript file.

@slavaleleka when you have some free time, please take a look.

**Comment by TPS** [comment]
> We'll need to scan all bunny CDN cnames and check if the domain hosts Fathom's javascript file.

Is it possible to check the _contents_ of the file while doing this?  If they're already randomizing domain names, then *file*names (& even content, similar to [how _other_ malware does](https://en.m.wikipedia.org/wiki/Polymorphic_code)) is likely.
